server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -1877,26 +1872,57 @@ private List<File> copyAndroidResourceFolders(String platform) throws ExtenderEx
                                                       .collect(Collectors.toList());
     }
 
+
+    // get extra packages (for aapt2) from the 'package' attribute in the AndroidManifest
+    // of the gradle dependencies. only get extra packages from aar dependencies which
+    // have a res folder
+    private List<String> getExtraPackagesFromGradlePackages() throws ExtenderException {
+        Set<String> extraPackages = new HashSet<String>();
+        try {
+            for (File f : gradlePackages) {
+                if(f.getName().endsWith(".aar")) {
+                    File res = new File(f, "res");
+                    File androidManifest = new File(f, "AndroidManifest.xml");
+                    if (res.exists() && androidManifest.exists()) {
+                        String am = FileUtils.readFileToString(androidManifest);
+                        Pattern p = Pattern.compile(".*package=\"(.*?)\".*", Pattern.MULTILINE | Pattern.DOTALL);
+                        Matcher m = p.matcher(am);
+                        if (m.matches()) {
```

Comment:
Possible improvement would be to actually parse the xml instead of using a regex. Is it worth it?

Reply:
I think this is fine for now. If it becomes an issue, we can do it later.